circle.yml

```diff
@@ -13,10 +31,13 @@ jobs:
           when: always
       - store_test_results:
           path: ~/junit
-  modules:
+  modules-jdbc-test:
     steps:
       - checkout
-      - run: ./gradlew check -x testcontainers:check -x selenium:check
+      - run:
+          command: ./gradlew jdbc-test:check
+          environment:
+            TZ: "/usr/share/zoneinfo/ETC/UTC"
```

Comment:
Adding `TZ` env var so that the Oracle JDBC driver doesn't cause an error. This is the first time we're testing the Oracle module on Circle CI 2.0, and it seems the first time that we've had to do work around this problem.

We _could_ include the timezone setting as a workaround inside the `OracleContainer` class, but I really think this would be the wrong place, as it's the driver that needs a timezone to be available. People using Oracle ought to be aware of this already or capable of resolving it.